extensions/localPublish/src/index.ts

```diff
@@ -52,6 +51,17 @@ const isLocalhostUrl = (matchUrl: string) => {
 const isSkillHostUpdateRequired = (skillHostEndpoint?: string) => {
   return !skillHostEndpoint || isLocalhostUrl(skillHostEndpoint);
 };
+
+const stringifyError = (error: any): string => {
+  if (error instanceof Error) {
+    return `${error.name} - ${error.message}`;
+  } else if (typeof error === 'object') {
+    return JSON.stringify(error, Object.getOwnPropertyNames(error));
+  } else {
+    return error;
```

Comment:
how about return error.toString()?

Reply:
if error is instanceof Error or typeof string, it can return toString(), but if it's an normal object, toString will throw error.

Reply:
I mean the your default condition, line 61, should be error.toString() right? since your function are supposed to return a string

Reply:
done.